### PR TITLE
doc: add news topics for 9.0.5

### PR DESCRIPTION
**Pull request by komainu8**

### CODE REVIEW

File: doc/source/news.rst [hunk 1]
```diff
@@ -7,6 +7,46 @@
...
+
+  * This feature is used only when the target shard is large enough.
+  * Large enough threshold is 10000 by default.
+
+* [:doc:`/reference/normalizers`] Added new option ``unify_to_katakana`` for ``NormalizerNFKC100``.
+
+  * This option normalize hiragana to katakana.
+  * For example, ``ゔぁゔぃゔゔぇゔぉ`` normalize to ``ヴァヴィヴヴェヴォ``.
+
+* Supported Amazon Linux 2.
```
Comment (kou) [inline doc/source/news.rst]:
```suggestion
* Added support for Amazon Linux 2.
```

Comment (komainu8) [reply doc/source/news.rst]: Thank you for your suggestion.

File: doc/source/news.rst [hunk 2]
```diff
@@ -7,6 +7,70 @@
...
+
+* [:doc:`reference/commands/logical_range_filter`] Stop an optimization by the number of required records.
+
+  * This feature is used only when the target shard is large enough.
+  * Large enough threshold is 10000 by default.
+
+* [:doc:`/reference/normalizers`] Added new option ``unify_to_katakana`` for ``NormalizerNFKC100``.
+
+  * This option normalize hiragana to katakana.
+  * For example, ``ゔぁゔぃゔゔぇゔぉ`` normalize to ``ヴァヴィヴヴェヴォ``.
```
Comment (kenhys) [inline doc/source/news.rst]: `For example, xxx is normalized to xxx` is natural?

File: doc/source/news.rst [hunk 3]
```diff
@@ -7,6 +7,70 @@
...
+
+* [:doc:`reference/commands/select`] Added support drilldowns for a slice parameter.
+
```
Comment (kenhys) [inline doc/source/news.rst]: `Added drilldowns support as a slice parameter` or `Added support for drilldowns as a slice parameter` ?

File: doc/source/news.rst [hunk 4]
```diff
@@ -7,6 +7,70 @@
...
+* [:doc:`reference/commands/select`] Added support columns for a slice parameter.
+
+* [:doc:`reference/commands/select`] Added support for accessing ``_score`` in initial stage for slice parameter.
+
+* [:doc:`/reference/functions/highlight_html`], [:doc:`reference/functions/snippet_html`] Improve that extract a keyword from an expression of before executing a slice when we specify the slice parameter.
+
+* Added support for index search for nested equal like XXX.YYY.ZZZ == AAA.
+
```
Comment (kenhys) [inline doc/source/news.rst]: markup ``XXX.YYY.ZZZ == AAA`

File: doc/source/news.rst [hunk 5]
```diff
@@ -7,6 +7,70 @@
...
+* Fixed wrong parameter name in ``table_create`` tests. [GitHub#1000][Patch by yagisumi]
+
+* Fixed a bug that drilldown label is empty when a drilldown command is executed by ``command_version=3``. [GitHub#1001][Reported by yagisumi]
+
+* Fixed build error for Windows package on MinGW.
+
+* Fixed install missing COPYING for Windows package on MinGW.
+
+* Fixed a bug that don't higlight when specifing non-test query as highlight target keyword.
```
Comment (kenhys) [inline doc/source/news.rst]: typo highlight

**Comment by komainu8** [comment]
@kenhys Thank you for your comments.
I've fixed them.

File: doc/source/news.rst [hunk 6]
```diff
@@ -7,6 +7,72 @@
...
+* [:doc:`/reference/normalizers`] Added new option ``unify_to_katakana`` for ``NormalizerNFKC100``.
+
+  * This option normalize hiragana to katakana.
+  * For example, ``ゔぁゔぃゔゔぇゔぉ`` is normalized to ``ヴァヴィヴヴェヴォ``.
+
+* [:doc:`reference/commands/select`] Added drilldowns support as a slices parameter.
+
+* [:doc:`reference/commands/select`] Added columns support as a slices parameter.
+
+* [:doc:`reference/commands/select`] Improved that we can reference ``_score`` in the initial stage for slices parameter.
```
Comment (kou) [inline doc/source/news.rst]: `refer`.

File: doc/source/news.rst [hunk 7]
```diff
@@ -7,6 +7,72 @@
...
+
+* Fixed wrong parameter name in ``table_create`` tests. [GitHub#1000][Patch by yagisumi]
+
+* Fixed a bug that drilldown label is empty when a drilldown command is executed by ``command_version=3``. [GitHub#1001][Reported by yagisumi]
+
+* Fixed build error for Windows package on MinGW.
+
+* Fixed install missing COPYING for Windows package on MinGW.
+
+* Fixed a bug that don't highlight when specifing non-test query as highlight target keyword.
```
Comment (kou) [inline doc/source/news.rst]: test -> text

**Pull request merged**